test-app/config/ember-try.js

```diff
@@ -1,7 +1,9 @@
 'use strict';
 
+// eslint-disable-next-line n/no-unpublished-require
```

Comment:
Can't we fix it in ESLint configuration? Like telling ESLint that this file is not published?

Reply:
I've just disabled the rule for these specific cases. It was only actually needed in the node-tests anyway